superchain/configs/sepolia/tbn.toml

```
@@ -8,8 +8,9 @@ batch_inbox_addr = "0xFf00000000000000000000000000000000042069"
 canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
-fjord_time = 1723204838  #Saturday June 8 2024 10:47:18 AM UTC
-granite_time = 1723545055 #Monday June 12 2024 1:57:35 PM UTC
+fjord_time = 1723204838  # Fri Aug 09 2024 12:00:38 UTC
+granite_time = 1723545055 # Tue Aug 13 2024 10:30:55 UTC
+holocene_time = 1732633200 # Tue Nov 26 15:00:00 UTC 2024
```

Note that the Sepolia activation has already passed over a week ago. Did you locally set this with an override on all your nodes?

If that's intentional, you probably want to receive all future hardfork from Holocene onwards, so better just set the `superchain_time`.

```suggestion
superchain_time = 1732633200 # Holocene - Tue Nov 26 15:00:00 UTC 2024
```

---

Setting `superchain_time` simply means that you inherit the activation times that are set for the superchain automatically. That also works for CGT chains.

---

But again, the Sepolia activation has already passed. Do you intentionally retroactively set the Sepolia Holocene activation time because you locally activated last week with overrides?